chat/tasks.py

```diff
@@ -27,8 +26,7 @@ def handle_puzzle_solved(puzzle_id, answer_text):
     try:
         puzzle.chat_room.archive_channels()
         msg = f"{puzzle.name} has been solved with {answer_text}!"
-        puzzle.chat_room.get_service().announce(msg)
-        puzzle.chat_room.send_message(msg)
+        puzzle.chat_room.send_and_announce_message_with_embedded_urls(msg, puzzle)
```

Comment:
Do we actually need to include the embed here? Wondering if it's gonna feel a little spammy / unnecessary, what do you think?